baybe/recommenders/pure/base.py

```diff
@@ -182,6 +182,8 @@ def _recommend_with_discrete_parts(
             or self.allow_recommending_already_measured,
         )
 
+        # TODO: Introduce new flag to recommend batches larger than the search space
```

Comment:
No, this is one of the issues we've already discussed earlier an never addressed – Julian just ran into the problem. Julian implemented a (partial?) fix to it but I asked him to remove it again since it needs to be properly tested and requires a separate PR.

Here his original commit and you can also see the TODO note at the bottom that already existed prior to this PR:
<img width="1247" alt="image" src="https://github.com/user-attachments/assets/a31b2589-b269-4ec4-82dd-1f3d82189265">

We definitely need to fix this rather soon. Perhaps you can take care of it while I'm away?

Reply:
In order to proceed, I'll close and open an issue instead: #369